sdk/playwrighttesting/microsoft-playwright-testing/src/common/debugLogger.ts

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import debug from "debug";
```

Comment:
Unless there's a reason not to, you should use `@azure/logger` here like this example shows: https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/template/template/src/logger.ts

Then, customers can take advantage of:
- Automatic sanitization of sensitive information
- Ability to configure a custom logger and redirect output
- Ability to configure log levels via the AZURE_LOG_LEVEL environment variable

Using the `@azure/logger` core library also eliminates the need to depend on `debug`

Reply:
Since `@azure/logger` is also based on debug, we have moved our logging to `@azure/logger`.

Reply:
Excellent, thank you so much!